libminifi/include/c2/C2Protocol.h

```diff
@@ -75,7 +75,7 @@ class C2Protocol : public core::Connectable {
   /**
    * Determines if we are connected and operating
    */
-  virtual bool isRunning() {
+  virtual bool isRunning() const {
```

Comment:
`virtual` -> `override`

Reply:
Updated in a20c3a130d80bd73e4383b8161e23c52c77fb1fd